src/vxworks/mod.rs

```diff
@@ -65,24 +66,19 @@ pub type uid_t = ::c_ushort;
 pub type gid_t = ::c_ushort;
 pub type sigset_t = ::c_ulonglong;
 pub type key_t = ::c_long;
-pub type shmatt_t = ::c_ulong;
-
-pub type mqd_t = ::c_int;
 
 pub type nfds_t = ::c_uint;
-pub type nl_item = ::c_int;
 pub type stat64 = ::stat;
 
 pub type pthread_key_t = ::c_ulong;
 
 // From b_off_t.h
 pub type off64_t = ::c_longlong;
-pub type off_t64 = ::c_longlong;
 
 // From b_BOOL.h
-pub type BOOL = ::c_int; // excuse me what
+pub type BOOL = ::c_int;
```

Comment:
What is the layout of `_Bool` in C and `bool` in C++ for this target ? 

What does `sizeof(_Bool)` and `sizeof(bool)` report ?

Reply:
BOOL is a type we used in vxWorks, it is different than _Bool or bool.